bubble_sort.rb

```diff
@@ -2,11 +2,13 @@ module SortingSuite
   # Bubble sort class uses the classic bubble sort algorithm to sort an array
   class BubbleSort
     def should_swap?(array, index)
```

Comment:
you might consider moving these two methods `should_swap?` and `swap` below the `sort` method, since the `sort` method is the most important method in your class and also part of the class' public interface. (public interface meaning that the other classes that might use the `BubbleSort` class should only be calling the `sort` method.)